sample/kotlin/src/main/kotlin/com/datadog/android/sample/NavActivity.kt

```diff
@@ -36,6 +44,7 @@ class NavActivity : AppCompatActivity() {
     }
 
     override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        crashAndGetExceptionMessage()
```

Comment:
You're crashing all the time here. I'm not sure you want to commit this.

Reply:
Just a playground like mentioned before, nothing to be worry about.